src/components/usa-link/usa-link.spec.js

```diff
@@ -42,4 +42,9 @@ describe("progressively enhanced usa-link component", async () => {
   it("should render link with component markup", () => {
     expect(getInsideLink().className).toContain("usa-link");
   });
+
+  it("should render only one <a> tag", () => {
```

Comment:
This test successfully captures the current bug on `develop`. If you add it on develop it should fail with a `2 does not equal 1`